client/src/components/SideNav/index.js

```diff
@@ -39,6 +39,7 @@ class SideNav extends Component {
         <div id="filterDivider">Filters</div>
         <button> <a href='/dashboard'>All Household Events</a> </button>
         <button> <a href='/dashboard?view=myevents'>My Events</a> </button>
+        <button> <a href='/dashboard?view=assigned'>Events Assigned To Me</a> </button>
```

Comment:
Can we remove "Events" before "Assigned to Me"

Reply:
yes that is a good suggestion